Kudu.Services.Web/DebugConsole/Default.cshtml

```diff
@@ -1,186 +1,186 @@
-﻿@{

```

Comment:
can you fix this file as well. i don`t think you touch every line of the file.  it is hard to see what is your changes.